File: public/naya-20160520101611/C/sherlock.c
// Name : Sherlock
#include <stdio.h>
int main() {
	// input:
	// n : row number
	// m : column number
	// city_map : hold the representation of the city as a table
	int n, m, crime_row_number, crime_col_number, idx, jdx;
	char city_map[100][100], tmp[100];
	scanf("%d %d", &n, &m);
	for(idx = 0; idx< n; idx++) {
		scanf("%s", &tmp[0]);
		for(jdx = 0;jdx<m;jdx++) {
			city_map[idx][jdx] = tmp[jdx];
		}
	}


	// write your code here
	// store your results in `crime_row_number`,`crime_col_number`

	// output
	printf("%d %d", crime_row_number, crime_col_number);
	return 0;
}

File: public/nootan-20160520092127/C_SHARP/sherlock.cs
// Name : Sherlock
using System;
namespace Application {
	class sherlock {
		public static void Main(string[] args) {
			// input:
			// n : size
			// m : size
			// map : ratio
			int idx, jdx;
					string[] elements = (Console.ReadLine()).Split(" ");
		int n = Convert.ToInt32(elements[0]);
		int m = Convert.ToInt32(elements[1]);
		char[][] map = new char[][];
		string map_elements = "";
		for(idx=0;idx<n;idx++){
			map_elements = (Console.ReadLine()).Split(" ");
			for(jdx=0;jdx<m;jdx++){
				map[idx][jdx] = map_elements[jdx];
			}
		}

			// write your code here
			// store your results in `row`,`column`

			// output
					Console.WriteLine(row+" "+column);

	}	
	}}